### PR TITLE
openfst 1.8.2 thrax 1.3.8

### PR DESCRIPTION
**Pull request by Porkepix**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by bayandin** [comment]
A revision bump for `thrax` is required. Let's even try to bump its version

**Comment by bayandin** [comment]
`thrax` build fails with:

```
  ==> make
  thraxcompiler --input_grammar=byte.grm --output_far=byte.far
  Evaluating rule: kBytes
  ERROR: GenericRegister::GetEntry: dlopen(StringFst-function.so, 0x0001): tried: 'StringFst-function.so' (no such file), '/usr/local/lib/StringFst-function.so' (no such file), '/usr/lib/StringFst-function.so' (no such file), '/private/tmp/thrax-test-20220204-28835-2bnxuj/grammars/StringFst-function.so' (no such file), '/usr/local/lib/StringFst-function.so' (no such file), '/usr/lib/StringFst-function.so' (no such file)
  FATAL: Check failed: "(output) != (nullptr)" file: ./../include/thrax/evaluator.h line: 634
```

**Comment by chenrui333** [comment]
> `thrax` build fails with:
> 
> ```
>   ==> make
>   thraxcompiler --input_grammar=byte.grm --output_far=byte.far
>   Evaluating rule: kBytes
>   ERROR: GenericRegister::GetEntry: dlopen(StringFst-function.so, 0x0001): tried: 'StringFst-function.so' (no such file), '/usr/local/lib/StringFst-function.so' (no such file), '/usr/lib/StringFst-function.so' (no such file), '/private/tmp/thrax-test-20220204-28835-2bnxuj/grammars/StringFst-function.so' (no such file), '/usr/local/lib/StringFst-function.so' (no such file), '/usr/lib/StringFst-function.so' (no such file)
>   FATAL: Check failed: "(output) != (nullptr)" file: ./../include/thrax/evaluator.h line: 634
> ```

Do we have any upstream issue tracking this test failure?

**Comment by chenrui333** [comment]
Looks like 1.3.8 works in my local. Kicked off a new run.

**Comment by chenrui333** [comment]
cc @Homebrew/core

**Comment by danielnachun** [comment]
A missing file like that error could be due to a race condition.  Maybe try deparallelizing the build?

File: Formula/thrax.rb [hunk 1]
```diff
@@ -32,10 +31,8 @@ class Thrax < Formula
   fails_with gcc: "5"
 
   # Fix -flat_namespace being used on Big Sur and later.
-  patch do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/03cf8088210822aa2c1ab544ed58ea04c897d9c4/libtool/configure-big_sur.diff"
-    sha256 "35acd6aebc19843f1a2b3a63e880baceb0f5278ab1ace661e57a502d9d78c93c"
-  end
+  # patch ref, https://raw.githubusercontent.com/Homebrew/formula-patches/03cf8088210822aa2c1ab544ed58ea04c897d9c4/libtool/configure-big_sur.diff
+  patch :DATA
```
Comment (carlocab) [inline Formula/thrax.rb:35]: When none of our patches for the `-flat_namespace` bug apply, we usually just regenerate `configure` with `autoreconf --force --install --verbose`.

Comment (chenrui333) [reply Formula/thrax.rb:35]: cool, I will give a try.

Comment (chenrui333) [reply Formula/thrax.rb]: @carlocab Should be good to go.

**Review by carlocab — APPROVED** [review]
Thanks!

**Pull request closed without merging**